mock/mock_test.go

```diff
@@ -1260,7 +1260,7 @@ func Test_Arguments_Bool(t *testing.T) {
 func Test_WaitUntil_Parallel(t *testing.T) {
 
 	// make a test impl object
-	var mockedService *TestExampleImplementation = new(TestExampleImplementation)
+	var mockedService = new(TestExampleImplementation)
```

Comment:
Infact you don't need the `:=`

Reply:
`var mockedService = new(TestExampleImplementation)`
and 
`mockedService := new(TestExampleImplementation)`
doesn't matter

Reply:
Yeah, you've refactored a lot with convention, so suggested. BTW i'm curious whether we can remove vendor.

Reply:
not sure, that I understand correctly this part `and could also share context behind main.go file change`

- vendor - yeah, we can remove it.. or add in gitignore